### PR TITLE
chore: merge changes from pressbooks-book

### PR DESCRIPTION
**Pull request by greatislander**
- [x] Update dependencies as necessary
- [x] Configure continuous integration GitHub Action
- [x] Configure changelog/release GitHub Action

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ dev, main ]
+  pull_request:
+    branches: [ dev, main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '16' ]
```
Comment (SteelWagstaff) [inline .github/workflows/ci.yml:13]: @greatislander I don't think our build tools are compatible with Node 16 yet (sigh). See https://github.com/pressbooks/pressbooks-malala/runs/6281068108?check_suite_focus=true and https://github.com/pressbooks/pressbooks-build-tools/issues/283.

Comment (greatislander) [reply .github/workflows/ci.yml:13]: @SteelWagstaff noted.

**Comment by greatislander** [comment]
@SteelWagstaff can you give me permissions to manage this repository and https://github.com/pressbooks/aetna?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (SteelWagstaff) [reply .github/workflows/ci.yml]: I could be wrong about what was happening in those failed builds we were running into when I tried to bump node to 16 in other repos a while back though. I saw some weird stuff in build tools CI pipeline realted to peer dependencies and humanmade's eslint-config-react-app: https://github.com/pressbooks/pressbooks-build-tools/runs/7571285106?check_suite_focus=true that I was able to resolve by adding --legacy-peerd-deps in this commit: https://github.com/pressbooks/pressbooks-build-tools/pull/282/commits/876738ec0cfbb31db1bd94b051b5d48e36c7c6fc. See https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh. Not probably a great long term decision, I know, but I saw some promising discussion in humamade's repo about what they want to do: https://github.com/humanmade/coding-standards/issues/252. Maintaining and caring for the build tools repo itself is still a little over my head, tbh, but I'm trying to learn.

**Comment by SteelWagstaff** [comment]
@greatislander Should be done. Let me know if you can merge now?

**Review by SteelWagstaff — APPROVED** [review]
:+1:

**Pull request merged**